### DISCUSSION TITLE
How to go to another page from a button click?

### DISCUSSION THREAD

**Discussion by matt-grain** [reflex-dev/reflex#170] [Q&A]
Looks like a stupid question but... what would be the python code on the `on_click` event to load another page (like a `pc.link` does thru href)

Thanks !

**Comment by picklelo** [comment]
You can use the `pc.redirect` server-side event for this: https://pynecone.io/docs/events-reference/server-side

(It is hidden away in the docs we will highlight it better in an example since it's a common use case)

**Comment by FHU-yezi** [comment]
> Looks like a stupid question but... what would be the python code on the `on_click` event to load another page (like a `pc.link` does thru href)
> 
> Thanks !

Maybe we want to run some server-side logic and choose a page to jump, such as jump to dashboard page if you are already logged in, otherwise, jump to login page.

**Reply by picklelo** [reply]
Yup, I think `pc.redirect` can handle this case since it's run within an event handler.
